projects/plugins/jetpack/tests/php/core-api/wpcom-endpoints/WPCOM_REST_API_V2_Endpoint_Memberships_Test.php

```diff
@@ -356,6 +356,133 @@ public function test_create_product_with_remote_error() {
 		$this->assertErrorResponse( 'dummy_error', $response, 500 );
 	}
 
+	/**
```

Comment:
It doesn't seem like we have true integration tests for the wpcom-endpoints, just mocked unit tests. Let me know if I'm missing them.